download-and-preprocessing.kts

```diff
@@ -97,4 +105,28 @@ fun rename(dirString: String, type: String) {
 fun snakeToCamel(snake: String): String {
     return snake.split("-")
         .joinToString("") { it.capitalize() }
```

Comment:
I believe `capitalize` was deprecated at some point, but the replacement is also abhorrently ugly :P